Makefile

```diff
@@ -33,72 +33,65 @@ WGET := $(shell which wget 2>/dev/null)
 RUST := $(shell which rustup 2>/dev/null)
 CROSS := $(shell which cross 2>/dev/null)
 DOCKER := $(shell which docker 2>/dev/null)
-ASTILECTRON_BUILDER := $(shell which astilectron-bundler 2>/dev/null)
 
-DESTDIR := /
-INSTALL_PREFIX := usr/bin
-LIBRARY_PREFIX := usr/lib
-SHARE_PREFIX := usr/share
+INSTALL_PREFIX := "/usr"
+bindir := $(INSTALL_PREFIX)/bin
```

Comment:
Question: I notice this is lowercased - what does that mean? Asking to understand better

Reply:
I don't have a strong opinion. I noticed in the [GNU Make manual](https://www.gnu.org/prep/standards/html_node/Directory-Variables.html) they use these variable names. In [CMake](https://cmake.org/cmake/help/latest/module/GNUInstallDirs.html) they use capital ones. I think it is advisable to use standard names but I don't have a strong opinion on the casing.

Reply:
I changed to using capital letters.